submissions/unabyband/task-js-dom/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <link rel="stylesheet" href="css/main.css">
+        <title>unabyband DOM</title>
+    </head>
+    <body>
+        <header>
+            <a href="index.html"><div class="home-link"></div></a>
+            <span class="inner-text">Just a little visualisation about color temperature and white balance.</span>
+        </header>
+        <div class="main-container">
+            <nav class="sidebar">
+                <div class="side-menu inner-text" id="nav-menu">  
+                    <a href="#" class="side-menu-item" id="tab-0">Clear Sky</a>
```

Comment:
Change `id` to` data-tab=`

Reply:
Fixed. Sorry, but I just didn't know about "data attributes" in html at that time.